addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -595,7 +595,7 @@ def _import_fill_invoice_line_values(self, tree, xpath_dict, invoice_line_form,
         elif net_price_unit is not None:
             price_unit = (net_price_unit + rebate) / basis_qty
         elif price_subtotal is not None:
-            price_unit = (price_subtotal + allow_charge_amount) / billed_qty
+            price_unit = (price_subtotal + allow_charge_amount) / (billed_qty or 1)
```

Comment:
Isn't the issue also a possibility on the other cases?

Reply:
Not really, the basis quantity is defined as 
`basis_qty = 1`
and then reassigned as 
`basis_qty = float(basis_quantity_node.text) or 1` (after https://github.com/odoo/odoo/commit/bfc83df736596a32f6009ad673194db20f18dec8)
It seems we don't allow the `basis_qty` to be 0. On the other hand `billed_qty` is used for the resulting invoice line quantity so `0` needs to be allowed

Reply:
Indeed